### PR TITLE
Expose more conntrack-related error as metrics

### PR DESCRIPTION
**Pull request by yiningou**

### CODE REVIEW

**Comment by yiningou** [comment]
Expose more conntrack-related error as metrics.
Please review my codes. Thanks!
/assign jingyuanliang
/assign MrHohn

**Comment by jingyuanliang** [comment]
I'm thinking if we should still permute all metrics like this as the list grows, or switch to a map potentially with an allowlist to capture all metrics. @MrHohn thoughts?

**Comment by MrHohn** [comment]
> I'm thinking if we should still permute all metrics like this as the list grows, or switch to a map potentially with an allowlist to capture all metrics. @MrHohn thoughts?

In this case I won't expect the list to grow in a significant way - these are the conntrack stats that exist for years. So I'm fine with anything simple here.

File: pkg/metrics/collector/conntrack.go [hunk 1]
```diff
@@ -81,13 +97,26 @@ func parseHeader(line string) (*conntrackIndices, error) {
...
 		case "drop":
 			indices.drop = i
+		case "early_drop":
+			indices.earlyDrop = i
+		case "search_researt":
+			indices.searchRestart = i
 		}
 	}
-	if indices.insertFailed == -1 || indices.drop == -1 {
+	if indices.found == -1 || indices.invalid == -1 ||
```
Comment (MrHohn) [inline pkg/metrics/collector/conntrack.go:116]: Looks like this won't work as we didn't explicitly assign the other fields `-1` initially?

**Comment by jingyuanliang** [comment]
/lgtm
wait for @MrHohn to approve.

File: pkg/metrics/collector/conntrack.go (hunk 1 above)
Comment (yiningou) [reply pkg/metrics/collector/conntrack.go:124]: Done. Set all the indices to `-1` before this judgment.

**Review by MrHohn — APPROVED** [review]
Thanks!
/lgtm

**Pull request merged**